txpipe/utils/calibrators.py

```diff
@@ -367,8 +370,14 @@ def apply(self, g1, g2, subtract_mean=True):
         """
 
         if subtract_mean:
-            g1 = (g1 - self.c[0]) / (1 + self.K)
-            g2 = (g2 - self.c[1]) / (1 + self.K)
+            Nmask = dec > -25.0
```

Comment:
Perhaps we can make this split dec an option as well? Then if someone for some reason wants to revert to analyzing everything at once they can just set this dec cut to like -90 or something.